### ISSUE TITLE
Telemetry: Provide a way to opt-out for PnP Telemetry

### ISSUE THREAD

**Issue by estruyf** [pnp/sp-dev-fx-controls-react#475]
#### Category
- [x] Enhancement
- [ ] Bug
- [ ] Question

#### Version

Please specify what version of the library you are using: [latest]

#### Expected / Desired Behavior / Question

Provide a way to opt-out for PnP telemetry.

**Comment by estruyf** [comment]
This is available in the latest beta.

<img width="434" alt="Screenshot 2020-01-29 22 17 28" src="https://user-images.githubusercontent.com/2900833/73399355-60f88600-42e7-11ea-9699-9b3607eba636.png">
<img width="501" alt="Screenshot 2020-01-29 22 19 02" src="https://user-images.githubusercontent.com/2900833/73399356-60f88600-42e7-11ea-8433-a95ff068605f.png">
<img width="435" alt="Screenshot 2020-01-29 22 17 44" src="https://user-images.githubusercontent.com/2900833/73399357-61911c80-42e7-11ea-8706-bea9edae8f79.png">

**Comment by eduardpaul** [comment]
Hello @estruyf and thank you very much for addressing this! :)

**Comment by nbelyh** [comment]
Was this merged in 2.1.0?  If you call that code on the startup:
```
const telemetry = PnPTelemetry.getInstance();
telemetry.optOut();
```
And then simply open the "Network" tab in Dev Tools, all telemetry calls seems to be still there:
![image](https://user-images.githubusercontent.com/528366/100277225-1ecdbe80-2f63-11eb-83fe-da34a5f8fefc.png)

Or are these some other telemetry calls from those that have been disabled... or maybe I am missing something?

**Comment by joelfmrodrigues** [comment]
@nbelyh I believe that is just the default telemetry from the workbench. Closing this issue, but please open a new one if you find issues with the telemetry from the controls.
